easybuild/easyconfigs/i/impi/impi-2017.1.132-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,35 @@
+name = 'impi'
+version = '2017.1.132'
+
+homepage = 'http://software.intel.com/en-us/intel-mpi-library/'
+description = """The Intel(R) MPI Library for Linux* OS is a multi-fabric message
+ passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for
+ Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification."""
+
+gccver = '5.4.0'
+binutilsver = '2.26'
```

Comment:
@valtandor I see no reason to use `gccver` and `binutilsver`, since they're only used in one place?

Just use:

```python
toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
```

Same applies to the other easyconfigs in this PR, where you define `comp`, just use:

```python
comp = ('GCC', '5.4.0-2.26')